wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/HtmlClassUtil.java

```diff
@@ -2,54 +2,60 @@
 
 import org.apache.commons.configuration.Configuration;
 
-
 /**
  * Provides an enum which can be used to add HTML class attribute values to components.
  *
  * <p>
- * Included are a set of helpers to attach icons. By default these will apply icons from the <a href="http://fontawesome.io/icons/">Font Awesome</a>
- * library included with the default theme. Each icon may be individually overridden using a configuration property. Each overrideable icon
- * includes documentation of the configuration parameter used to set the icon's HTML class attribute value. If the default icons are changed your
- * theme must include a way to ensure the new icon is available. This is safest if simply changing to an alternate
+ * Included are a set of helpers to attach icons. By default these will apply icons from the
+ * <a href="http://fontawesome.io/icons/">Font Awesome</a>
+ * library included with the default theme. Each icon may be individually overridden using a configuration property.
+ * Each overrideable icon includes documentation of the configuration parameter used to set the icon's HTML class
+ * attribute value. If the default icons are changed your theme must include a way to ensure the new icon is available.
+ * This is safest if simply changing to an alternate
  * <a href="http://fontawesome.io/icons/">Font Awesome</a> icon.
  * </p>
  *
  * @author Mark Reeves
  * @since 1.2.0
  */
 public class HtmlClassUtil {
+
 	/**
 	 * The configuration instance.
 	 */
 	private static final Configuration CONFIG = Config.getInstance();
 
 	/**
-	 * The help icon HTML class. Defaults to a <a href="http://fontawesome.io/icons/">Font Awesome</a> question mark icon.
```

Comment:
Did these really need wrapping? I'm pretty sure our rules say 150 chars per line, what's your IDE set to?